tests/testAutopas/tests/selectors/ContainerSelectorTest.cpp

```diff
@@ -32,11 +32,29 @@ TEST_F(ContainerSelectorTest, testSelectAndGetCurrentContainer) {
   }
 }
 
+void getStatus(const std::array<double, 3> &bBoxMin, const std::array<double, 3> &bBoxMax, const double cutoff,
+               autopas::ContainerSelector<Particle, FPCell> &containerSelector, std::vector<Particle> &ListInner,
+               std::vector<Particle> &ListHalo, std::vector<Particle> &ListHaloVerletOnly) {
+  for (auto iter = containerSelector.getCurrentContainer()->begin(autopas::IteratorBehavior::ownedOnly); iter.isValid();
+       ++iter) {
+    ListInner.push_back(*iter);
+  }
+  for (auto iter = containerSelector.getCurrentContainer()->begin(autopas::IteratorBehavior::haloOnly); iter.isValid();
+       ++iter) {
+    if (autopas::utils::inBox(iter->getR(), autopas::ArrayMath::sub(bBoxMin, std::array<double, 3>{cutoff}),
+                              autopas::ArrayMath::add(bBoxMax, std::array<double, 3>{cutoff}))) {
```

Comment:
for readability, you could move the array calculations to separate variables like in lines 76,77.